plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataimport/CommonElementImporter.java

```diff
@@ -690,13 +692,16 @@ protected void parseParameter(final FBNetworkElement block) throws TypeImportExc
 			throw new TypeImportException(Messages.ImportUtils_ERROR_ParameterNotSet);
 		}
 
+		if (name.contains(".") && block instanceof final TypedSubApp tsa) { //$NON-NLS-1$
+			parseVarConfigs(tsa, name);
+		}
```

Comment:
I you move this down to the getinterfacElement part you do not need to change the getInterfaceelement method